content/en/agent/docker/apm.md

```diff
@@ -53,12 +53,14 @@ List of all environment variables available for tracing with the Docker Agent:
 | `DD_APM_DD_URL`            | Datadog API endpoint where traces are sent. For Datadog EU site set `DD_APM_DD_URL` to `https://trace.agent.datadoghq.eu` |
 | `DD_APM_RECEIVER_PORT`     | Port that the Datadog Agent's trace receiver listens on. Default value is `8126`.                                          |
 | `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers).                             |
-| `DD_APM_IGNORE_RESOURCES`   | A comma-separated list of endpoints and resources that the Agent should ignore (i.e. health checks endpoints).            |
-| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions.                                                                          |
+| `DD_APM_IGNORE_RESOURCES`  | Configure resources for the agent to ignore. Format should be comma seperated\* regular expressions. i.e. `"GET /ignore-me,(GET|POST) /and-also-me"` |
+| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions. Format should be comma seperated\* instances of `<SERVICE_NAME>|<OPERATION_NAME>=1`. i.e. `my-express-app|express.request=1,my-dotnet-app|aspnet_core_mvc.request=1` |
 | `DD_APM_ENV`               | Sets the default [environment][2] for your traces.                                                                        |
 | `DD_APM_MAX_EPS`           | Sets the maximum APM events per second.                                                                                   |
 | `DD_APM_MAX_TPS`           | Sets the maximum traces per second.                                                                                       |
 
+**\****No additional characters such as spaces or newlines are allowed between commas*
```

Comment:
```suggestion
**Note**: No additional characters such as spaces or newlines are allowed between commas
```

Reply:
I think you handled this better with the info block in your PR which is now merged. I think I'll just scrap this